src/js/lib/monster.ui.js

```diff
@@ -3490,6 +3522,55 @@ define(function(require) {
 	}
 	ui.monthpicker = monthpicker;
 
+	/**
+	 * Transforms a field into a jQuery UI Spinner element
+	 * @param  {jQuery} $target  Input to transform
+	 * @param  {Object} options  List of options
+	 * @return {jQuery}          jQuery UI Spinner instance
+	 */
+	function spinner($target, pOptions) {
+		if (!($target instanceof $)) {
+			throw TypeError('"$target" is not a jQuery object');
+		}
+		if (!_.isPlainObject(pOptions)) {
```

Comment:
Is `pOptions` mandatory or optional? I see it is named `pOption` so assume its optional but the jsdoc does not reflect that (`options` is not between `[]`).

If it is optional, you should update your check to see if it is not both `undefined` and a plain object at the same time.